pkg/asset/agent/image/baseiso.go

```diff
@@ -91,6 +105,29 @@ func (i *BaseIso) Dependencies() []asset.Asset {
 	}
 }
 
+func (i *BaseIso) checkReleasePayloadBaseISOVersion(r Release, archName string) error {
+	logrus.Debugf("Checking release payload base ISO version")
+
+	// Get current release payload CoreOS version
+	payloadRelease, err := r.GetBaseIsoVersion(archName)
+	if err != nil {
+		return err
```

Comment:
Since the only consequence of this failing is that we can't tell whether to print a warning, maybe if this fails we should print a warning and return nil.